asyncpg/connect_utils.py

```diff
@@ -442,6 +458,21 @@ def _parse_connect_dsn_and_args(*, dsn, host, port, user,
     else:
         sslmode = SSLMode.disable
 
+    if sslcert is None:
```

Comment:
An explicit `SSLContext` passed to `connect()` is considered to be of higher priority than the environment, so let's move this block inside `if isinstance(ssl, (str, SSLMode)):` above.